main.go

```diff
@@ -94,7 +95,6 @@ func main() {
 		Desc:   "Throttle",
 		EnvVar: "THROTTLE"})
 
-	//TODO can we use custom headers
 	messageTypeEndpointsMap := map[string]string{
```

Comment:
this can be simplified if we expect the publisher to send the correct value (the plural), we get rid of a lot of complexity this way